### PR TITLE
Port phytosian

### PR DESCRIPTION
**Pull request by ThatLing**
I did this a while ago when porting both phytosian and preternis. I finished phytosian but never finished preternis (fucking powereating losers). Most (read: All) numbers are pulled out of my ass, balancing might be horribly broken. :)

 Adds phytosian (podpeople/plantpeople) as a roundstart race and makes them behave more like yog phytosian.

### CODE REVIEW

**Review by Amelia0010 — APPROVED** [review]
we can always rebalance the numbers if need be

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -37,6 +37,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/burnmod = 1		// multiplier for burn damage
 	var/coldmod = 1		// multiplier for cold damage
 	var/heatmod = 1		// multiplier for heat damage
+	var/acidmod = 1     // multiplier for acid damage // yogs - Old Plant People
 	var/stunmod = 1		// multiplier for stun duration
 	var/punchdamagelow = 0       //lowest possible punch damage
 	var/punchdamagehigh = 9      //highest possible punch damage
```
Comment (simonset55) [inline code/modules/mob/living/carbon/human/species.dm:43]: This is too low, it should be set to 69, have you ever been punched by a plant before? no? well i can tell you it hurts

File: config/game_options.txt [hunk 2]
```diff
@@ -425,7 +425,7 @@ ROUNDSTART_RACES plasmaman
 #ROUNDSTART_RACES skeleton
 #ROUNDSTART_RACES zombie
 #ROUNDSTART_RACES slime
-#ROUNDSTART_RACES pod
+ROUNDSTART_RACES pod
```
Comment (simonset55) [inline config/game_options.txt:428]: Now this, is pod racing!

File: yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+	attack_sound = 'sound/weapons/slice.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	burnmod = 2
+	heatmod = 1.5
+	coldmod = 1.5
+	acidmod = 2
+	speedmod = 0.33
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/plant
+	disliked_food = MEAT | DAIRY
+	liked_food = VEGETABLES | FRUIT | GRAIN
```
Comment (simonset55) [inline yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm:20]: Just like how humans eat humans, plantpeople eat plants

File: yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+		return
+	var/turf/T = get_turf(H)
+	if(!T)
+		return
+
+	var/light_level = 0
+	var/light_msg //for sending "low light!" messages to the player.
+	var/light_amount = T.get_lumcount() //how much light there is in the place, affects receiving nutrition and healing
+	if(istype(H.loc, /obj/mecha) || istype(H.loc, /obj/machinery/clonepod))
+		//let's assume the interior is lit up
```
Comment (simonset55) [inline yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm:57]: no

File: yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+			return 0
+
+		H.adjustBruteLoss(-0.5*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustFireLoss(-0.5*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustToxLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
+		H.adjustOxyLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
+		H.reagents.remove_reagent(chem.id, chem.metabolization_rate * REAGENTS_METABOLISM)
+		return 1
+
+	if(chem.id == "sugar")
```
Comment (simonset55) [inline yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm:209]: omg this is shitcode, why have a set way for the race to react to sugar, when you could just add diabetes as a disease and give all plant people that. Fucking idiots learn to simplify your code

File: yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+	var/last_light_level = 0
+	var/last_light_message = -STATUS_MESSAGE_COOLDOWN
+	var/last_plantbgone_message = -STATUS_MESSAGE_COOLDOWN
+
+/datum/species/pod/before_equip_job(datum/job/J, mob/living/carbon/human/H)
+	to_chat(H, "<span class='info'><b>You are a Phytosian.</b> Born on the core-worlds of G-D52, you are a distant relative of a vestige of humanity long discarded.</span>")
+	to_chat(H, "<span class='info'>Symbiotic plant-cells suffuse your skin and provide a protective layer that keeps you alive, and affords you regeneration unmatched by any other race.</span>")
+	to_chat(H, "<span class='info'>Darkness is your greatest foe. Even the cold expanses of space are lit by neighbouring stars, but the darkest recesses of the station's interior may prove to be your greatest foe.</span>")
+	to_chat(H, "<span class='info'>Heat and cold will damage your epidermis far faster than your natural regeneration can match. You can communicate with other phytosians use <b>:P</b>.</span>")
+	to_chat(H, "<span class='info'>For more information on your race, see https://wiki.yogstation.net/index.php?title=Phytosian</span>")
```
Comment (simonset55) [inline yogstation/code/modules/mob/living/carbon/human/species_types/plantpeople.dm:34]: Make it link to meatspin instead

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 7]
```diff
@@ -597,12 +597,13 @@
...
 	//DAMAGE//
+	var/damagemod = (dna && dna.species) ? dna.species.acidmod : 1 // yogs - Old Plant People
 	for(var/obj/item/bodypart/affecting in damaged)
 		affecting.receive_damage(acidity, 2*acidity)
 
 		if(affecting.name == BODY_ZONE_HEAD)
 			if(prob(min(acidpwr*acid_volume/10, 90))) //Applies disfigurement
-				affecting.receive_damage(acidity, 2*acidity)
+				affecting.receive_damage(acidity, 2*acidity*damagemod) // yogs - Old Plant People
 				emote("scream")
```
Comment (simonset55) [inline code/modules/mob/living/carbon/human/human_defense.dm:607]: I also scream when i notice that i have been shaved and i'm bald

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 8]
```diff
@@ -1,3 +1,4 @@
+// yogs - This file is mirrored to plantpeople.dm
 /datum/species/pod
 	// A mutation caused by a human being ressurected in a revival pod. These regain health in light, and begin to wither in darkness.
 	name = "Podperson"
```
Comment (simonset55) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm:4]: This is pretty racist actually, it would be like calling a human a "wombperson"

**Review by simonset55 — COMMENTED** [review]
yes

**Pull request merged**